lte/gateway/python/magma/pipelined/tests/app/subscriber.py

```diff
@@ -24,8 +24,10 @@
 
 from magma.subscriberdb.sid import SIDUtils
 
-SubContextConfig = namedtuple('ContextConfig', ['imsi', 'ip', 'table_id'])
-
+SubContextConfig = namedtuple('ContextConfig', ['imsi', 'ip', 'ambr', 'table_id'])
+AmbrConfig = namedtuple('AmbrConfig', ['max_bandwidth_ul', 'max_bandwidth_dl'])
+#default_ambr_config = AmbrConfig(max_bandwidth_ul=1000000, max_bandwidth_dl=1000000)
```

Comment:
so should it be none or AmbrConfig?

Reply:
Since none of the tests use this, it can be none. I will clean this up.